.gitignore

```diff
@@ -1,4 +1,3 @@
-src/**/*.js
```

Comment:
what for?

Reply:
I've accidentally compiled TS to JS in IDEA and spent 30 minutes working out why my code crashes and I get stack traces of JS files. This `.gitignore` entry hid the problem and should be removed in my opinion. Doesn't give any value really.

Reply:
Context: given an option `ts-node` decided to load JS files over TS.

Reply:
I hit the same problem this morning, and lost a similar amount of time, I ran `yarn compile` to reproduce an issue jenkins was having.

hmm, sounds reasonable. @geek-soft any objection?

Reply:
Not really, we can try it however real issue is that `ts-node` loads JS files and we should solve that first.

Reply:
This thing happened to me once too.

@geek-soft 
I agree that ts-node should give preference to TS files over JS, but I think at least part of the problem is that build output files are put in src directory. If we maintain good configuration hygiene and keep our input and output separate, we won't need to depend on the ts-node behaviour as much. Also, all we would need in .gitignore to handle this is a single entry for the build output directory.